src/wp-includes/block-bindings/post-meta.php

```diff
@@ -17,18 +17,15 @@
  *                           Example: array( "key" => "foo" ).
  * @return mixed The value computed for the source.
```

Comment:
Nit: the param description is missing. You can copy from the other source.

Reply:
I've added it [here](https://github.com/WordPress/wordpress-develop/pull/6079/files).